.devcontainer/cuda11.8-conda/devcontainer.json

```diff
@@ -5,17 +5,17 @@
     "args": {
       "CUDA": "11.8",
       "PYTHON_PACKAGE_MANAGER": "conda",
-      "BASE": "rapidsai/devcontainers:24.12-cpp-cuda11.8-mambaforge-ubuntu22.04"
+      "BASE": "rapidsai/devcontainers:25.02-cpp-cuda11.8-mambaforge-ubuntu22.04"
```

Comment:
This PR targets 24.12. Why have all the dependencies been changed to 25.02?

Reply:
This appears to be an attempt at https://github.com/rapidsai/cudf/issues/13593, an issue I filed a while back. @the-gates-of-Zion I can help work with you on this, if you are interested. This should target `branch-25.02`, so I will retarget the PR.